orders/models.py

```diff
@@ -71,3 +72,51 @@ def cancel_order(self):
 
     def get_order_total_price(self):
         return self.price_per_day * (self.end_date - self.start_date).days
+
+    def get_order_status(self):
+        if self.status == StatusOptions.Pending:
+            return "Pending"
+        elif self.status == StatusOptions.Approved:
+            return "Approved"
+        elif self.status == StatusOptions.Canceled:
+            return "Canceled"
+        elif self.status == StatusOptions.OnGoing:
+            return "Ongoing"
+        return "Finished"
+
+    @staticmethod
+    def get_capacity_of_daycare_in_dates_range(daycare_id, start_datetime, end_datetime):
+        number_of_days = (end_datetime - start_datetime).days
+        capacity_per_day: list[int] = [0] * number_of_days
+        start_date = datetime.date(year=start_datetime.year, month=start_datetime.month, day=start_datetime.day)
+        end_date = datetime.date(year=end_datetime.year, month=end_datetime.month, day=end_datetime.day)
+        relevant_orders = Order.objects.filter(daycare_id=daycare_id,
+                                               status__in=['A', 'O'],
+                                               end_date__gte=start_date,
+                                               start_date__lte=end_date)
+
+        for order in relevant_orders:
+            for day in range(number_of_days):
+                current_date = order.start_date + datetime.timedelta(days=day)
+                if current_date < start_date:
+                    continue
+                elif current_date > end_date:
+                    break
+                else:
+                    capacity_per_day[day] = capacity_per_day[day] + 1
+
+        return capacity_per_day
+
+    def are_order_dates_available(self):
```

Comment:
test?

Reply:
I made "test_order_is_approvable_according_to_daycare_capacity" for this method. Should I test it in another way?

Reply:
Is it testing this function alone? 
if it does, it deserves a name change :)

Reply:
The only thing this function does is check if specific dates are available due to capacity. the test you wrote deal with the whole process your application does. What we need for this function is a **unit** test, as it tests this unit specifically, without dependencies from other processes.